pom.xml

```diff
@@ -166,7 +166,7 @@
     <joda.version>2.9.3</joda.version>
     <jodd.version>3.5.2</jodd.version>
     <jsr305.version>1.3.9</jsr305.version>
-    <libthrift.version>0.9.3</libthrift.version>
+    <libthrift.version>0.12.0</libthrift.version>
```

Comment:
The highest version of libfb303 is 0.9.3. In Spark3.0, libfb303 is also 0.9.3.

Reply:
Good suggestion! I did not notice this problem.